cli/internal/transformer/transformer.go

```diff
@@ -62,6 +69,9 @@ func (t *RecordTransformer) TransformSchema(sc *arrow.Schema) *arrow.Schema {
 		if _, ok := mdMap["cq:extension:primary_key"]; ok && t.removePks {
 			mdMap["cq:extension:primary_key"] = "false"
```

Comment:
Not related to the functionality of this fix but why not delete the key from the map instead?
```suggestion
			delete(mdMap, "cq:extension:primary_key")
```